pkg/operator/controller/externaldns/pod.go

```diff
@@ -56,10 +56,11 @@ const (
 	//
 	// AWS
 	//
-	awsAccessKeyIDEnvVar     = "AWS_ACCESS_KEY_ID"
-	awsAccessKeySecretEnvVar = "AWS_SECRET_ACCESS_KEY"
-	awsAccessKeyIDKey        = "aws_access_key_id"
-	awsAccessKeySecretKey    = "aws_secret_access_key"
+	awsCredentialEnvVarName  = "AWS_SHARED_CREDENTIALS_FILE"
+	awsCredentialsVolumeName = "aws-credentials"
+	awsCredentialsDir        = "/aws"
+	awsCredentialsFilePath   = "/aws/credentials"
+	awsSecretKey             = "credentials"
```

Comment:
```suggestion
	awsCredentialsMountPath        = defaultConfigMountPath
	awsCredentialsFileKey             = "credentials"
	awsCredentialsFileName = "aws-credentials"
	awsCredentialsFilePath = filepath.Join(awsCredentialsMountPath, awsCredentialsFileName)
```
To match what's done for other providers.

Reply:
@alebedev87 `throws error `filepath.Join(awsCredentialsMountPath, awsCredentialsFileName) (value of type string) is not constant`

Can I just put `	awsCredentialsFilePath   = awsCredentialsMountPath + "/" + awsCredentialsFileName`
`